welcome.py

```diff
@@ -1,32 +1,30 @@
 from telethon import events, errors
 
 
-WELCOME = {
-    -1001109500936:
-    'Hi and welcome to the group. Before asking any questions, **please** '
-    'read [the docs](https://docs.telethon.dev/). Make sure you are '
-    'using the latest version with `pip3 install -U telethon`, since most '
-    'problems have already been fixed in newer versions.',
-
-    -1001200633650:
-    'Welcome to the off-topic group. Feel free to talk, ask or test anything '
-    'here, politely. Check the description if you need to test more spammy '
-    '"features" of your or other people\'s bots (sed commands too).'
-}
+WELCOME = 'Hi and welcome to the group. Before asking any questions, **please** '\
+          'read [the docs](https://docs.telethon.dev/). Make sure you are '\
+          'using the latest version with `pip3 install -U telethon`, since most '\
+          'problems have already been fixed in newer versions.'
```

Comment:
This should probably stay as a dict so it supports adding more chats if necessary

Reply:
Well the old one didn't had a check to see if the chat Id exists or not in the dict so I assumed there won't be any chats added